python/syntax.lark

```diff
@@ -52,6 +54,8 @@ listdef: LIST "[" NAME "]"
 tupleval: "(" value ("," value)* ")"
 listval: "[" (value ("," value)*)? "]"
 recordval: "{" (record_entry (";" | "\n"))* record_entry (";" | "\n")? "}"
+imported_value: NAME "?" NAME
```

Comment:
Also maybe we should make all imports records. By merging imports and variables, we can avoid confusing name clashes that currently are allowed, like

```js
import whatever
let whatever = { a: 3; b: 4 }
```